resources/import.sql

```diff
@@ -70,6 +70,7 @@ AVG_ROW_LENGTH = 4096;
 CREATE TABLE IF NOT EXISTS npwd_sellout_listings (
   id int(11) NOT NULL AUTO_INCREMENT,
   identifier varchar(255) DEFAULT NULL,
+  player varchar(255) DEFAULT NULL,
```

Comment:
What is player representing here, identity name, or?

Edit: Ah, now that I see the type is a string, this makes me think it's the identity name. We might want to rename that column to something more similar to that.

2nd Edit: Now I see you are grabbing playerName from the GetPlayerName native. Might be better to rename this to username.